PyInstaller/utils/hooks/hookutils.py

```diff
@@ -725,26 +718,11 @@ def is_module_version(module_name, comparison_name, module_version):
     Examples
     ----------
         # Test whether the local version of Sphinx is 1.3.x or newer.
+        >>> from operator import ge
         >>> from PyInstaller.utils.hooks.hookutils import is_module_version
-        >>> is_module_version('sphinx', '>=', '1.3.1')
+        >>> is_module_version('sphinx', ge, '1.3.1')
         True
     """
-    # Dictionary mapping passed comparison names to private tuple method names.
-    comparison_to_method_name = {
-        '>=': '__ge__',
-        'ge': '__ge__',
-        '<=': '__le__',
-        'le': '__le__',
-        '>':  '__gt__',
-        'gt': '__gt__',
-        '<':  '__lt__',
-        'lt': '__lt__',
-    }
```

Comment:
@leycec Oh, you've gone further than I intended. I would have kept the string-based call as you have had (`is_module_version('sphinx', '>=', '1.3.1')`. I'd only changed a) this mapping to map to `operator.ge` etc, and b) using this mapped operator (like it is done in the new code around line 738).

Sorry for not being expressive enough on this.